modular_skyrat/modules/customization/modules/clothing/clothing.dm

```diff
@@ -188,3 +188,6 @@
 
 /obj/item/clothing/under/rank/civilian/head_of_personnel/suit/skirt
 	mutant_variants = NONE
+
+/obj/item/clothing/under/costume/draculass
```

Comment:
Does not show on digitigrade characters, but does not show the SUIT placeholder either. intended?